File: app/controllers/output_controller.rb
class OutputController < UIViewController
  extend IB

  outlet :timeLabel, UILabel
  outlet :backgroundImageView, UIImageView

  def viewDidLoad
    super

    @appearance = Appearance.sharedClient

  end

  def viewDidLayoutSubviews
    super

    updateAppearance

  end

  def updateAppearance

    self.view.backgroundColor = @appearance.backgroundColor
    @backgroundImageView.image = @appearance.backgroundImage
    @timeLabel.setFont(UIFont.fontWithName(@appearance.fontName, size:(@appearance.fontScale * self.view.frame.size.width)))
    @timeLabel.textColor = @appearance.textColor

  end

end

File: Countout/OutputController.swift
import UIKit

class OutputController: UIViewController {
    var appearance = Appearance.sharedClient
    
    @IBOutlet weak var countdownView: CountdownView!

    override func viewDidLoad() {
        super.viewDidLoad()
        
        updateAppearance()
    }
    
    func updateAppearance() {
        countdownView.backgroundColor = appearance.backgroundColor
        countdownView.backgroundImage = appearance.backgroundImage
        countdownView.setFont(name: appearance.fontName, size: appearance.fontScale)
        countdownView.textColor = appearance.textColor
    }
}